### PR TITLE
Change jwt encrypt to default disabled

### PR DESCRIPTION
**Pull request by gemmai95**
**What is the context of this PR?**

Change default jwt encrypt to disabled to match other services.

Link to Trello card

Describe what you have changed and why, link to other PRs or Issues as appropriate.

**How to review**

Any notes for the reviewer  (include screenshots if appropriate).

### CODE REVIEW

**Comment by gemmai95** [comment]
[SM137: EXT Send SM from To do List](https://trello.com/c/UU26Qrr6/76-sm137-ext-send-sm-from-to-do-list)

File: config.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Config:
     VERSION = os.getenv('VERSION', '0.1.2')
 
     SECURE_MESSAGING_DATABASE_URL = os.getenv(
-        'SECURE_MESSAGING_DATABASE_URL', 'postgresql://postgres:postgres@localhost:5432')
+        'SECURE_MESSAGING_DATABASE_URL', 'postgresql://postgres:postgres@localhost:6432')
```
Comment (AdamHawtin) [inline config.py]: If you're changing the default database port does the README need updating to reflect this?

Comment (gemmai95) [reply config.py]: Going to change this back. As this has been discussed to be part of a different PR.

File: config.py [hunk 2]
```diff
@@ -139,10 +139,12 @@ class DevConfig(Config):
     CLIENT_SECRET = os.getenv('CLIENT_SECRET', 'loginsecret')
     UAA_URL = os.getenv('UAA_URL', 'http://uaa-int.apps.devtest.onsclofo.uk')
     USE_UAA = int(os.getenv('USE_UAA', 1))
+    SM_JWT_ENCRYPT = os.getenv('SM_JWT_ENCRYPT', '1')
 
 
 class TestConfig(DevConfig):
     TESTING = True
+    SM_JWT_ENCRYPT = os.getenv('SM_JWT_ENCRYPT', '1')
```
Comment (AdamHawtin) [inline config.py]: Isn't this line already inherited from DevConfig now?

**Pull request merged**